### PR TITLE
chore: project addition by tedante #2123

### LINKED ISSUE #2123: chore: project addition by tedante
### Add a new project to the list

Add my project gsheet-api-helper and zoom-api-helper into Project hut

### Record

- [X] I have checked the existing [issues](https://github.com/priyankarpal/ProjectsHut/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/priyankarpal/ProjectsHut/blob/main/CODE_OF_CONDUCT.md)
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by tedante**
## Related Issue

closes #2123 

## Description

add an project to DB/projects.json file

## Screenshots

no screenshots

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -4710,5 +4710,22 @@
...
     ]
+  },
+  {
+    "github_username": "tedante",
+    "Projects": [
+      {
+        "github_url": "https://github.com/justcodebeast/zoom-api-helper",
+        "title": "zoom-api-helper",
+        "description": "Zoom API Helper is a Node.js package that provides a helper for simplifies managing meet.",
+        "tech": ["axios", "typescript"]
```
Comment (priyankarpal) [inline DB/projects.json:4721]: remove  `axios`
```suggestion
        "tech": ["typescript"]
```

File: DB/projects.json [hunk 2]
```diff
@@ -4710,5 +4710,22 @@
...
+      },
+      {
+        "github_url": "https://github.com/justcodebeast/gsheet-api-helper",
+        "title": "gsheet-api-helper",
+        "description": "Google Spreadsheet API Helper is a Node.js package that provides a helper for simplifies the process of accessing, retrieving and updating data from Gsheet using the Gsheet API.",
+        "tech": ["axios", "typescript", "jsonwebtoken"]
```
Comment (priyankarpal) [inline DB/projects.json:4727]:
```suggestion
        "tech": ["typescript"]
```

File: DB/projects.json [hunk 3]
```diff
@@ -4710,5 +4710,22 @@
...
+    "github_username": "tedante",
+    "Projects": [
+      {
+        "github_url": "https://github.com/justcodebeast/zoom-api-helper",
+        "title": "zoom-api-helper",
+        "description": "Zoom API Helper is a Node.js package that provides a helper for simplifies managing meet.",
+        "tech": ["axios", "typescript"]
+      },
+      {
+        "github_url": "https://github.com/justcodebeast/gsheet-api-helper",
```
Comment (priyankarpal) [inline DB/projects.json:4724]: add your project

File: DB/projects.json (hunk 1 above)
Comment (tedante) [reply DB/projects.json:4721]: sure, thank you for reviewing @priyankarpal

File: DB/projects.json [hunk 4]
```diff
@@ -4710,5 +4710,22 @@
         ]
       }
     ]
+  },
+  {
+    "github_username": "tedante",
+    "Projects": [
+      {
+        "github_url": "https://github.com/justcodebeast/zoom-api-helper",
```
[the remark this answers is not in the document]
Comment (tedante) [reply DB/projects.json:4718]: that is my project in community,
is it not allowed if the repo placed not in my primary account?

[the remark this answers is not in the document]
Comment (priyankarpal) [reply DB/projects.json:4718]: ook got it.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by priyankarpal** [comment]
Hey @tedante! 👋

For secure collaboration, remember to check out GitHub's commit signature verification feature. It helps ensure the authenticity of commits and prevents tampering.

Learn how it works: [GitHub Docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

Happy coding! 😄

**Pull request merged**